### PR TITLE
Brightness AVX support

### PR DESCRIPTION
**Pull request by r-abishek**
Adds brightness AVX implementation for tensor host

### CODE REVIEW

File: src/include/cpu/rpp_cpu_common.hpp [hunk 1]
```diff
@@ -2069,6 +2069,106 @@ inline RppStatus custom_convolve_image_host(T* srcPtr, RppiSize srcSize, U* dstP
 
 // Compute Functions for RPP Tensor API
 
+inline RppStatus compute_brightness_48_host(__m256 *p, __m256 *pBrightnessParams)
```
Comment (rrawther) [inline src/include/cpu/rpp_cpu_common.hpp:2072]: looks like a duplicate definition for the same function below

File: src/include/cpu/rpp_cpu_simd.hpp [hunk 2]
```diff
@@ -909,24 +909,24 @@ inline RppStatus rpp_store24_f32pln3_to_f32pkd3_avx(Rpp32f *dstPtr, __m256 *p)
     return RPP_SUCCESS;
 }
 
-inline RppStatus rpp_load8_f32_to_f32_avx(Rpp32f *srcPtr, __m256 *p)
+inline RppStatus rpp_load16_f32_to_f32_avx(Rpp32f *srcPtr, __m256 *p)
 {
     p[0] = _mm256_loadu_ps(srcPtr);
+    p[1] = _mm256_loadu_ps(&srcPtr[8]);
```
Comment (rrawther) [inline src/include/cpu/rpp_cpu_simd.hpp:915]: please change it to use _mm256_loadu_ps(srcPtr+8)

**Review by rrawther — CHANGES_REQUESTED** [review]
@paveltc : Please run all unit-tests to make sure these functions work

**Comment by paveltc** [comment]
@rrawther @asalmanp @r-abishek This PR passes all unit tests for HOST.

**Comment by kiritigowda** [comment]
@r-abishek - Conflicts with TOT

File: src/include/cpu/rpp_cpu_simd.hpp (hunk 2 above)
Comment (r-abishek) [reply src/include/cpu/rpp_cpu_simd.hpp]: Done

**Comment by kiritigowda** [comment]
@paveltc let me know if this is good to merge?

**Review by rrawther — APPROVED** [review]
@paveltc to verify

**Pull request merged**